src/sentry/api/urls.py

```diff
@@ -1502,6 +1503,11 @@ def create_group_urls(name_prefix: str) -> list[URLPattern | URLResolver]:
         OrganizationMetricsExtractionRulesEndpoint.as_view(),
         name="sentry-api-0-organization-metrics-extraction-rules",
     ),
+    re_path(
+        r"^(?P<organization_id_or_slug>[^\/]+)/metrics-usage/$",
```

Comment:
Makes sense, I opted in for this way to make this API endpoint more robust, so that we can query it and get all the widgets and alerts where specific metric MRI is used (which doesn't have to come from span extraction rule).

We can also offer both options:
- if the `ruleId` is present in URL - API returns all the alerts and widgets for the rule
- if there is no `ruleId` - API returns all the alerts and widgets connected to MRIs from `metricMRIs` parameter

Reply:
I agree with @obostjancic here, and would add that MRIs are not user-facing entities, so we should not make them queriable to the user. Span metrics are the future, we should treat the endpoint as such: how can we infer usage of a span metric? As far as I'm concerned, MRI is an implementation detail at this point and should therefore not be exposed to the user.
There is an additional question here I think - is the ID really the identifer we want to use? I can imagine this being more idiomatic by querying by the span attribute, but I haven't fully thought this through at the moment.

Reply:
Span attribute might be tricky, since it can be the same for multiple projects inside the organization.

To me, it makes the most sense if we for now use the id of the config.

Reply:
After the discussion, it was concluded that the access pattern will be (org.slug, project.slug, span_attribute) in an URL